### PR TITLE
Update bootbox package.json

### PR DESCRIPTION
**Pull request by tiesont**
Updates package.json to reflect new repository structure, to wit:
- Un-minified files moved to `src` directory
- Individual files created in `src/locales` for each locale, allowing only those locales which are used to be included
- Minified files created in `dist` directory

# Pull request template for new library

Pull request for issue: #13280 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/makeusabrew/bootbox
* Official website (optional, not the repository): http://bootboxjs.com/
* NPM package url (optional): https://www.npmjs.com/package/bootbox
* License and its reference: MIT https://github.com/makeusabrew/bootbox/blob/master/LICENSE.md
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 213
  * Count of stars: 4565
  * Count of forks: 1035
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [ ] This lib was not found on cdnjs repo
* [ ] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by MattIPv4** [comment]
Resolves #13280

**Comment by tiesont** [comment]
Is there anything I need to do to get this pulled in?

**Comment by MattIPv4** [comment]
cc @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Looks like the file/directory structure was changed? Would you please mention each change in the commit message? (Instead of `Update package.json`)

**Comment by tiesont** [comment]
Are the changes I noted above sufficient, or do you need more than that?

If necessary, if I reverted the master branch to use the old directory structure, would cdnjs automatically update? Not a big deal to make that change, if so.

**Comment by MattIPv4** [comment]
@PeterDaveHello You might need to do the rebase to edit the commit message?

File: ajax/libs/bootbox.js/package.json [hunk 1]
```diff
@@ -16,13 +16,16 @@
       {
         "basePath": "",
         "files": [
-          "bootbox.js"
+          "src/*.js",
+          "src/*/*.js",
+          "dist/*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/bootbox.js/package.json]: I think you may want to use this?
```json
    "fileMap": [
      {
        "basePath": "src",
        "files": [
          "**/*"
        ]
      },
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
```

**Pull request closed without merging**